### PR TITLE
feat: add remix guide and starter repo

### PR DESCRIPTION
**Pull request by Ashutosh-Bhadauriya**
Adds remix guide along with starter

### CODE REVIEW

**Comment by Ashutosh-Bhadauriya** [comment]
@FlxMgdnz @lfleischmann could you please take a look when you get a chance 🙂 , it would be good to announce our remix guide asap. I think its good to merge, have tested everything btw and proofread too, but just in case

File: quickstarts/fullstack/remix.mdx [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+
+export { register, Hanko };
+```
+
+
+## Add `<hanko-auth>` component
+
+The `<hanko-auth>` web component adds a login interface to your app. This code defines a `HankoAuth` component in the `login` route. 
+
+It uses the register function from `hanko.client.ts` to register Hanko with the provided URL. On successful registration, it sets the hanko state. When hanko is set, it sets up a callback for when the authentication flow is completed, redirecting the user to the dashboard.
```
Comment (lfleischmann) [inline quickstarts/fullstack/remix.mdx]: > " ... to register Hanko with the provided URL."

I feel that, technically, this is a bit inaccurate. What it does is register a custom element with the custom element registry of the browser.

File: quickstarts/fullstack/remix.mdx [hunk 2]
```diff
@@ -0,0 +1,359 @@
...
+```tsx utils/hanko.client.ts
+import { register, Hanko } from "@teamhanko/hanko-elements";
+
+export { register, Hanko };
+```
+
+
+## Add `<hanko-auth>` component
+
+The `<hanko-auth>` web component adds a login interface to your app. This code defines a `HankoAuth` component in the `login` route.
```
Comment (lfleischmann) [inline quickstarts/fullstack/remix.mdx]:
```suggestion
The `<hanko-auth>` web component adds a login interface to your app. The following code example defines a `HankoAuth` component in the `login` route. 
```

File: quickstarts/fullstack/remix.mdx (hunk 1 above)
Comment (lfleischmann) [inline quickstarts/fullstack/remix.mdx]: >  it sets the hanko state

Again, I feel this is too vague. As a user, I wouldn't even know what "the hanko state is". Maybe give a short explain what `Hanko` actually is in this case?

File: quickstarts/fullstack/remix.mdx [hunk 3]
```diff
@@ -0,0 +1,359 @@
...
+export default Dashboard
+```
+
+## Customize component styles
+
+You can customize the appearance of `hanko-auth` and `hanko-profile` components using CSS variables and parts. Refer to our [customization guide](/guides/hanko-elements/customize-appearance).
+
+## Securing routes
+
+To add JWT verification with Hanko in your Remix application, we're using [jose library](https://www.npmjs.com/package/jose). However, you're free to choose any other suitable library.
```
Comment (lfleischmann) [inline quickstarts/fullstack/remix.mdx]:
```suggestion
To add JWT verification with Hanko in your Remix application, we're using the [jose](https://www.npmjs.com/package/jose) library. However, you're free to choose any other suitable library. 
```

File: resources/examples.mdx [hunk 4]
```diff
@@ -50,6 +50,29 @@ description: Example apps showcasing integration of Hanko with your favorite fra
   >
     Full source code availabe on our GitHub
   </Card>
+  <Card
```
Comment (lfleischmann) [inline resources/examples.mdx:53]: Now that the number of examples is growing, would it make sense to order them alphabetically?

File: quickstarts/fullstack/remix.mdx [hunk 5]
```diff
@@ -0,0 +1,359 @@
...
+
+  if (!userID) {
+    return null;
+  }
+
+  return userID;
+}
+```
+
+This is how you can use it in your routes
```
Comment (lfleischmann) [inline quickstarts/fullstack/remix.mdx]:
```suggestion
This is how you can use it in your routes:
```

File: resources/examples.mdx (hunk 4 above)
Comment (Ashutosh-Bhadauriya) [reply resources/examples.mdx:53]: I have added them based on popularity, and what are most searched on the docs. So, the priority is full-stack frameworks (sorted based on popularity) and then frontend frameworks in the same way. wdyt?

File: quickstarts/fullstack/remix.mdx (hunk 1 above)
Comment (Ashutosh-Bhadauriya) [reply quickstarts/fullstack/remix.mdx]: Modified it to this, please let me know

The `<hanko-auth>` web component adds a login interface to your app. The following code example defines a `HankoAuth` component in the `login` route. 

It uses the register function from `hanko.client.ts` to register `<hanko-auth>` component with the brower's [CustomElementRegistry](https://developer.mozilla.org/de/docs/Web/API/CustomElementRegistry). When the authentication flow is completed, a callback is set up to redirect the user to the dashboard.

**Comment by Ashutosh-Bhadauriya** [comment]
Thanks @lfleischmann, have addressed the changes. Please let me know

File: resources/examples.mdx (hunk 4 above)
Comment (lfleischmann) [reply resources/examples.mdx:53]: I think it makes finding what I'm looking for ("do they have an example for X") more difficult and that alphabetical order is more predictable. 

I'll resolve this here though, we can open another issue for that.

**Review by lfleischmann — APPROVED** [review]

**Pull request merged**